qp_woltka/tests/test_woltka.py

```diff
@@ -449,11 +449,12 @@ def test_woltka_syndna_to_array(self):
             '#SBATCH -N 1\n',
             '#SBATCH -n 8\n',
             '#SBATCH --time 12:00:00\n',
-            '#SBATCH --mem 190g\n',
+            '#SBATCH --mem 5g\n',
```

Comment:
Just to be sure, this value jumped from 190gb to 5gb. Is this okay?

Reply:
Yes, tested in qiita-rc and it worked fine.